src/pages/home/ReportScreen.js

```diff
@@ -432,7 +432,7 @@ function ReportScreen({
                                         policies={policies}
                                     />
                                 ) : (
-                                    <ReportFooter shouldDisableCompose />
```

Comment:
Is there any reason for removing `shouldDisableCompose` ?

Reply:
For this requirement https://github.com/Expensify/App/issues/27779#issuecomment-1735522530

> If you could update your proposal so that send button won't be disabled during chat loading, but be disabled only after clicking on it

Reply:
thanks! 🙇🏼